docs/body.md

```diff
@@ -538,6 +541,19 @@ changes will still be caught without this option enabled, it is
 recommended to leave notify_on_state_change disabled if your
 OpenStack environment is very large.
 
+#### Network and Router Info Events
+
+The OpenStack Infrastructure ZP pulls neutron events for Networks and Routers. However,
+if Neutron is not configured properly to send those events, they cannot be retrieved.
+If you are missing Info events, checks your OpenStack environment's 
+/etc/neutron/neutron.conf. It should have the following:
+
+```
```

Comment:
Ah, ok.  Was going on context clues, and I guess I got the wrong end of the stick there.  Simple fix.  Will update in a sec.